packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -66,10 +67,16 @@ export class Subscribers extends WithHttp implements ISubscribers {
     });
   }
 
+  async deleteCredentials(subscriberId: string, providerId: string) {
+    return await this.http.delete(
+      `/subscribers/${subscriberId}/credentials/${providerId}`
+    );
+  }
+
   async unsetCredentials(subscriberId: string, providerId: string) {
```

Comment:
Shall we remove this `unsetCredentials` method after this new `deleteCredentials` method?

Reply:
maybe its better to deprecate